codec-native-quic/src/main/c/netty_quic_quiche.c

```diff
@@ -665,8 +667,13 @@ static void netty_quiche_config_set_active_connection_id_limit(JNIEnv* env, jcla
 }
 
 static void netty_quiche_config_set_stateless_reset_token(JNIEnv* env, jclass clazz, jlong config, jbyteArray token) {
-    uint8_t* buf = (uint8_t*) (*env)->GetByteArrayElements(env, token, 0);
-    quiche_config_set_stateless_reset_token((quiche_config*) config, buf);
+    jbyte* buf = (*env)->GetByteArrayElements(env, token, NULL);
+    jsize buf_len = (*env)->GetArrayLength(env, token);
+    uint8_t* token_buf = (uint8_t*) malloc(buf_len * sizeof(uint8_t));
+    memcpy(token_buf, buf, buf_len);
```

Comment:
Why do we need the extra memcpy here ? Shouldn't the original code be fine as long as we add the missing release ?